stl/inc/cmath

```diff
@@ -616,12 +616,16 @@ _STD _Common_float_type_t<_Ty1, _Ty2> remquo(_Ty1 _Left, _Ty2 _Right, int* _Pquo
         return _CSTD FUN(static_cast<double>(_Left), _Arg2);                   \
     }
 
+// When the arguments are both floats or both long doubles, overload resolution prefers the
+// non-template overloads to the templates generated from this macro. The templates generated from
+// this macro are only selected by overload resolution when both arguments have integral type, or
+// when the types of the two arguments differ, in which case _Common_float_type_t is either double
+// or long double.

```

Comment:
It would be good, imo, to note that `long double` and `double` have the same representation on our implementation.

Additionally, if I remember correctly from our meeting, doesn't Intel use our STL with 80-bit `long double`?

Reply:
> It would be good, imo, to note that `long double` and `double` have the same representation on our implementation.

"Everyone" knows this, right? ;) I'll add "Note that `double` and `long double` have the same underlying representation on our supported platforms."

> Additionally, if I remember correctly from our meeting, doesn't Intel use our STL with 80-bit `long double`? Or did I misunderstand.

Yes they do, and I suspect they have problems with our `<cmath>`  - maybe they override the entire header? - but this change is immaterial. The templates generated from this macro pass values to a signature that takes `double`s, the only change I'm making is to convert the arguments to `double` explicitly instead of `implicitly`.